src/styles/_typography.scss

```diff
@@ -27,8 +42,18 @@ p {
   max-width: 65ch;
 }
 
+ul p,
+ol p {
+  margin-block-start: 0 !important;
+}
+
+p + ul,
+p + ol {
+  margin-block-start: var(--gcds-spacing-200) !important;
```

Comment:
This one I'm less confident about, but with the way we add `mt-400` to everything, I don't see another way to do it. Can we target lists following a paragraph in `anchor.js`? that way we could not have this css. And possibly delete lines 45-48 above too

Reply:
Removing the general change to top margin please and reassess if this is still needed.

Reply:
removed. But now there is too much space in between list items: Just for full transparency:
<img width="1169" alt="Capture d’écran, le 2023-02-09 à 15 17 49" src="https://user-images.githubusercontent.com/5230720/217927511-7a3dd11d-e388-4a90-83e6-69b851283386.png">

Reply:
Is there a way to remove list items being wrapped in p tags?
Or can we modify your code from above and do something like this to ensure there is no margin-bottom if the p is the last list item?

```
ul li p:last-child,
ol li p:last-child {
  margin-block-end: var(--gcds-spacing-0) !important;
```

Reply:
We could simplify to 

```css
li p:last-child {
  margin-block-end: var(--gcds-spacing-0) !important;
}
```

What do you think?

Reply:
Actually both css options won't work with nested lists. The last `<li>` will have margin, even if it isn't then end of a list. 

Now, we could add `mb-400` to ul and ol in `anchor.js`, but it won't fix nested lists, as all ul and ol would receive bottom margin.

Unless we also add some css like this:

```css
ol ol,
ul ul {
    margin-block-end: var(--gcds-spacing-0) !important;
}
```

Reply:
It's ok, we don't need it to work with nested lists since the lists here are not nested and for the lists that are nested within our components, the components should take care of that styling.

Reply:
Ok, I went with your css